e2e/js/js-cloud-server/yarn.lock

```diff
@@ -1497,27 +1497,27 @@ __metadata:
   linkType: hard
 
 "@devcycle/js-cloud-server-sdk@file:../../../dist/sdk/js-cloud-server::locator=e2e-js-cloud-server%40workspace%3A.":
-  version: 1.17.2
-  resolution: "@devcycle/js-cloud-server-sdk@file:../../../dist/sdk/js-cloud-server#../../../dist/sdk/js-cloud-server::hash=2335fe&locator=e2e-js-cloud-server%40workspace%3A."
+  version: 1.9.2
```

Comment:
why is this version being downgraded?

Reply:
dunno - local ref being weird? didn't touch the actual cloud sdk ref tho